ISSUE_TEMPLATE.md

```diff
@@ -1,9 +1,8 @@
-### What did you do?
-(Please include as many details as possible.)
+### Describe the problem and steps to reproduce it:
```

Comment:
I think a good compromise here would be to just change the first item as you have here and leave the rest as you suggested (maybe you could leave the `{please include as many details as possible)` bit too. 

I would have thought in general questions work better than cold headings (since they invite answers) - but that's just a hunch. We'll need to just get it in place on every repo and see if people use it, as that will be the best measure of it in practice.